<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>797</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Гиперкуб</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="GLWidget" name="widgetHyperCube" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,1,100,100">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="5">
       <widget class="QScrollBar" name="horizontalScrollBar_Y2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QScrollBar" name="horizontalScrollBar_Ch2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QScrollBar" name="horizontalScrollBar_Ch1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QScrollBar" name="horizontalScrollBar_X2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedControls">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QScrollBar" name="horizontalScrollBar_Y1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QScrollBar" name="horizontalScrollBar_X1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Канал</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Строки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Столбцы</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_19">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_20">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_21">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>slider1ValueChanged(int)</slot>
    <slot>sliderCh1ValueChanged(int)</slot>
    <slot>sliderCh2ValueChanged(int)</slot>
    <slot>sliderX1ValueChanged(int)</slot>
    <slot>sliderX2ValueChanged(int)</slot>
    <slot>sliderY1ValueChanged(int)</slot>
    <slot>sliderY2ValueChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalScrollBar_Ch1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderCh1ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Ch2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderCh2ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_X1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderX1ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_X2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderX2ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Y1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderY1ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Y2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetHyperCube</receiver>
   <slot>sliderY2ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>879</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>868</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Ch1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_19</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Ch2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_X1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_20</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_X2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>589</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Y1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_21</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Y2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>679</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
